lib/puppetserver/ca/action/generate.rb

```diff
@@ -62,19 +83,24 @@ def run(input)
           signer = SigningDigest.new
           return 1 if CliParsing.handle_errors(@logger, signer.errors)
 
+          if input['subject_alt_names'].empty?
+            subject_alt_names = munge_alt_names(puppet.settings[:subject_alt_names])
+          else
+            subject_alt_names = munge_alt_names(input['subject_alt_names'])
+          end
+
```

Comment:
I think it would be easier to read something like:
```
alt_names_to_munge = input['subject_alt_names'].empty? puppet.settings[:subject_alt_names] : input['subject_alt_names']
subject_alt_names = munge_alt_names(alt_names_to_munge)
```

Reply:
I went back and forth on this. I didn't like doing the munge in-line, but I'm also not sure about creating the other throwaway variable. But I'll change it if that seems better.

Reply:
I don't mind the temp variable, though I don't think we should have a 120 character ternary.

Reply:
The length was one of the main reasons I went this way instead. Even though I also don't love variables that are only assigned inside if blocks 🤷‍♀️